dwarf-rust/src/main/wasm/README.md

```diff
@@ -0,0 +1,6 @@
+# Attribution
+
+wasm-source-map.wasm was built from:
+
+https://github.com/chirino/wasm-source-map
```

Comment:
cool! maybe add a commit hash just so we know where this version comes from

Reply:
done.